rosa_hcp/rosa-hcp-sts-creating-a-cluster-ext-auth.adoc

```diff
@@ -0,0 +1,94 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="rosa-hcp-sts-creating-a-cluster-ext-auth"]
+= Creating ROSA with HCP clusters with external authentication
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-hcp-sts-creating-a-cluster-ext-auth
+
+toc::[]
+
+You can create {hcp-title-first} clusters that use an external authentication to issue your access tokens.
+
+[IMPORTANT]
+====
+Since it is not possible to upgrade or convert existing ROSA clusters to a {hcp} architecture, you must create a new cluster to use {hcp-title} functionality.
```

Comment:
Per ISG, do not use _since_ to mean _because_.
```suggestion
Because it is not possible to upgrade or convert existing ROSA clusters to a {hcp} architecture, you must create a new cluster to use {hcp-title} functionality.
```